src/prolly/chunker.rs

```diff
@@ -46,10 +46,10 @@ mod tests {
 
     #[test]
     fn test_hash_byte() {
-        const S: &str = "NaN";
```

Comment:
@aboodman, I wanted to get a sanity check on the changes here and above, which make the mask pattern into the N bits being set where N is the shift amount, versus just having the N + 1th bit set and no others.

Do we have any golden test data from elsewhere that we can use for this test?